### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by jaxaz**
This PR is:

- [x ] Adding a new domain

- [ x] I used the uncompressed size of the site
- [x ] I have included a link to the GTMetrix report
- [ x] The domain is in the correct alphabetical order
- [ ] This site is not a ultra lightweight site
- [ x ] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [x ] Check to confirm

```
- domain: col2in.ml
  url: https://col2in.ml
  size: 10.8 kb
  last_checked: 2022-06-20
```

GTMetrix Report: 
https://gtmetrix.com/reports/col2in.ml/sEQr3Dew/

### CODE REVIEW

**Comment by garritfra** [comment]
@kmutahar UM? 🤔 Though I think this is okay, since it's not just a contact page.

File: _data/sites.yml [hunk 1]
```diff
@@ -497,7 +497,7 @@
   url: https://cock.li/
   size: 474
   last_checked: 2022-01-29
-
+
```
Comment (garritfra) [inline _data/sites.yml:500]: This whitespace might have slipped in by accident. Could you remove it?

**Review by garritfra — CHANGES_REQUESTED** [review]

**Comment by kevquirk** [comment]
No, this is NSFW so not suitable for us either I'm afraid. Just look at some of the domains on offer.

**Comment by garritfra** [comment]
@kevquirk why do you think this is NSFW?

**Comment by kevquirk** [comment]
Look at the domains on offer for their service -

**Comment by kevquirk** [comment]
Sorry, I mis-read, I was looking at the commit info - cock.li was right above it and I thought it was this domain. Sorry!

I think this is fine to be added.

cock.li needs to be removed though.

**Comment by kmutahar** [comment]
@kevquirk cock.li while the name is not the best and may be seen as NSFW, the site content is not but is an email service.

**Comment by kevquirk** [comment]
@kmutahar look at the domains on offer. It's 100% NSFW.

**Review by garritfra — APPROVED** [review]

**Comment by garritfra** [comment]
@kevquirk @kmutahar fyi: removed the other domain.

99879d5faae8570619c573aa98a3a5b123fdbe69

**Pull request merged**